packages/build/src/plugins/child/utils.js

```diff
@@ -9,12 +14,11 @@ const { show } = require('./status')
 // Retrieve the `utils` argument.
 const getUtils = function ({ event, constants: { FUNCTIONS_SRC, INTERNAL_FUNCTIONS_SRC, CACHE_DIR }, runState }) {
   const build = getBuildUtils(event)
-  const utils = { build }
-  addLazyProp(utils, 'git', getGitUtils)
-  addLazyProp(utils, 'cache', getCacheUtils.bind(null, CACHE_DIR))
-  addLazyProp(utils, 'run', getRunUtils)
-  addLazyProp(utils, 'functions', getFunctionsUtils.bind(null, FUNCTIONS_SRC, INTERNAL_FUNCTIONS_SRC))
-  addLazyProp(utils, 'status', getStatusUtils.bind(null, runState))
+  const cache = getCacheUtils(CACHE_DIR)
+  const functions = getFunctionsUtils(FUNCTIONS_SRC, INTERNAL_FUNCTIONS_SRC)
+  const status = getStatusUtils(runState)
+  const utils = { build, cache, run, functions, status }
+  addLazyProp(utils, 'git', () => getGitUtils())
```

Comment:
`utils.git` returns a bunch of git-related information. That information is retrieved by making `git` calls, which is slow. In other words, accessing `utils.git` requires executing some (slow) code. This is in the contrast to the other utilities which just expose methods to the plugin authors. Therefore, we need to keep loading this one lazily.